concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -313,6 +314,25 @@ public Semver getVersion() {
         return new Semver(versionString);
     }
 
+    /**
+     * Get the list of smart contract modules in the given block.
+     * @param blockHash {@link Hash} of block at which the modules list is being retrieved.
+     * @return Parsed {@link ImmutableList} of {@link Hash}
+     * @throws Exception When the returned JSON is null.
+     */
+    public ImmutableList<Hash> getModuleList(final Hash blockHash) throws Exception {
```

Comment:
I think you want to return `ImmutableList<ModuleRef>`?

Reply:
done